### PR TITLE
Duo/tpv5 mirror mesh

### PR DESCRIPTION
**Pull request by daisy20170101**
I add the files for generating mirrored mesh for tpv5, which has been tested by me, for this issue
 https://github.com/SeisSol/SeisSol/issues/546
@NicoSchlw @Thomas-Ulrich should we remove the old version tpv5 mesh?

### CODE REVIEW

File: tpv5/tpv5_f200m.geo [hunk 1]
```diff
@@ -48,15 +48,15 @@ gmsh2gambit -i planar-anydip.msh -o planar-anydip.neu
 lc = 25e3;
 lc_fault = 200;
 
-Fault_length = 30e3;
-Fault_width = 15e3;
+Fault_length = 44e3;
+Fault_width = 22e3;
```
Comment (NicoSchlw) [inline tpv5/tpv5_f200m.geo:52]: The previous fault geometry was right, the new values belong to tpv105

File: tpv5/tpv5_f200_halfdom.geo [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+ Generate mirror mesh about fault plane with:
+
+ gmsh -3 tpv5_f200_halfdom.geo
+ pumgen tpv5_f200_halfdom.msh -s msh2
+ python mirrorMesh.py tpv5_f200_halfdom.xdmf 1 0
+*/
+
+lc= 5000;
+lc_fault = 200;
```
[the remark this answers is not in the document]
Comment (daisy20170101) [reply tpv5/tpv5_f200_halfdom.geo:45]: I added the domain size, fault size and free surface lc :-)

File: tpv5/generating_mirror_mesh.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+# Generate half a mesh using gmsh
+#for gmsh, see http://gmsh.info/#Download
+#gmsh -3 $prefix.geo
+
+# Convert the mesh from neu to hdf5 using pumgen
+# for pumgen, see https://github.com/SeisSol/PUMGen/wiki/How-to-compile-PUMGen
+pumgen $prefix.msh -s msh2
+
+#mirror the mesh using mirrorMesh.py (from here https://github.com/SeisSol/Meshing/tree/master/mirrorMesh/mirrorMesh.py)
+python mirrorMesh.py $prefix.xdmf 1 0
```
Comment (NicoSchlw) [inline tpv5/generating_mirror_mesh.sh:13]: Line 6 should be uncommented and I would use python3 in line 13. Apart from that it looks fine, I generated a mesh and everything worked well

Comment (daisy20170101) [reply tpv5/generating_mirror_mesh.sh:13]: Fixed. Thanks Nico!

**Comment by Thomas-Ulrich** [comment]
Maybe we can keep the geo file for reference and remove generating_the_mesh.sh
or remove both.

**Review by Thomas-Ulrich — APPROVED** [review]
LGTM

**Pull request merged**